### PR TITLE
Locale and Interval types are not global anymore

### PR DESCRIPTION
**Pull request by hydrognomik**
Tyring to fix issue #1472.

### CODE REVIEW

File: scripts/build/_lib/typings/typeScript.js [hunk 1]
```diff
@@ -10,6 +10,20 @@ const {
...
+  if (['Locale', 'Interval'].some(t => type.includes(t))) {
+    type = type.replace(/(Locale|Interval)/g, match => `dateFns.${match}`)
+  }
+
+  return type
+}
+
+const decoratedGetType = exportedTypesDecorator(getType)
+const decoratedGetParams = exportedTypesDecorator(getParams)
+const decorategetFPFnType = exportedTypesDecorator(getFPFnType)
```
Comment (dkozickis) [inline scripts/build/_lib/typings/typeScript.js]: Please correct casing and name - `decoratedGetFPFnType`?

**Comment by hydrognomik** [comment]
Hello. Can't import `Locale`. Should we declare it in `dateFns` module but exports from `'date-fns'`.

**Comment by dkozickis** [comment]
Yep, turns out it has to be done like that.

**Comment by dkozickis** [comment]
@leshakoss / @kossnocorp - appreciate eyes on from one of you.

**Pull request closed without merging**